verification/curator-service/ui/src/components/ViewCase.tsx

```diff
@@ -81,6 +81,15 @@ function ageRange(range?: { start: number; end: number }): string {
         : `${range.start}-${range.end}`;
 }
 
+function dateRange(range?: { start: string; end: string }): string {
```

Comment:
You could merge this and the ageRange function into one by having the inputs be string | number.

Reply:
no because the dates ar ecurrently returned with "-" as their separators and joining them with "-" without space as we do for ages is awkward, that's why I added a space in the formatting below, hard to catch I admit :)

Reply:
I was thinking the age could also be separated with " - ", but not a big deal.